packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -79,6 +80,7 @@ BuildRequires: %{?scl_prefix}rubygem(stomp)
 BuildRequires: %{?scl_prefix}rubygem(gettext_i18n_rails)
 BuildRequires: %{?scl_prefix}rubygem(apipie-rails) >= 0.5.14
 BuildRequires: %{?scl_prefix}rubygem(fx) < 1.0
+BuildRequires: %{?scl_prefix}rubygem(pg)
```

Comment:
Can you please drop the `BuildRequires: %{?scl_prefix}rubygem(pg)` in line 28? That was added as a workaround until Katello's gemfile was fixed.

Reply:
Done